### PR TITLE
chore/fix-ui-tests

### PR DESCRIPTION
**Pull request by pascal-brause**
## Description
This fixes some ui tests.
I already added a test plan change for circle ci that a branch with this name ("chore/fix-ui-tests") will always build the UI tests, too. So its easier to fix them.

Fixes:

test_0022_AppInformationFlow_faq()
test_0023_AppInformationFlow_contact()
test_0026a_AppInformationFlow_ErrorReportsShouldBeActiveForDebugAsDefault()
test_0024_AppInformationFlow_privacy()
test_0026_AppInformationFlow_ErrorReports()
test_0025_AppInformationFlow_terms()
test_0029_AppInformationFlow_ConfirmationScreen_ErrorReportDetailScreen()
test_0030_AppInformationFlow_ConfirmationScreen_HistoryScreen()
test_0028_AppInformationFlow_PrivacyScreen()
test_0027_AppInformationFlow_ErrorReportsStart()
test_0021_AppInformationFlow_about()
testDetailsSelectionOfLocationVisit()

### CODE REVIEW

**Comment by naveeddotio** [comment]
Small request: Could you please write the fixed tests in the description.

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENAUITests/ContactJournal/ENAUITests_07_ContactJournalUITests.swift [hunk 1]
```diff
@@ -256,23 +256,32 @@ class ENAUITests_07_ContactJournal: CWATestCase {
...
-		app.keys["0"].waitAndTap()
-		app.keys["4"].waitAndTap()
-		app.keys["2"].waitAndTap()
-		// Close keyboard.
-		app.waitAndTap()
-
-		// Wait for closing.
-
+		// iOS 15 changed the time picker to a wheel again.
+		if #available(iOS 15.0, *) {
```
Comment (marcussc) [inline src/xcode/ENA/ENAUITests/ContactJournal/ENAUITests_07_ContactJournalUITests.swift:272]: Love having the backwards comparability in here :)

File: src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift [hunk 2]
```diff
@@ -425,9 +425,8 @@ class ENAUITests_04a_ExposureSubmission: CWATestCase {
 		app.buttons[AccessibilityIdentifiers.ExposureSubmissionDispatch.findTestCentersButtonDescription].waitAndTap()
 	
 		// Check if safari was opened
-		let safariApp = XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
-		let runInForeground = safariApp.wait(for: .runningForeground, timeout: .long)
-		XCTAssertTrue(runInForeground)
+		XCTAssertTrue(XCUIApplication(bundleIdentifier: "com.apple.mobilesafari").wait(for: .runningForeground, timeout: .extraLong))
```
Comment (marcussc) [inline src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift:428]: I have a feeling this will never be not flaky 🙈

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift [hunk 3]
```diff
@@ -50,7 +50,6 @@ class ENAButton: DynamicTypeButton {
 	@IBInspectable var isInverted: Bool = false { didSet { applyStyle() } }
 	@IBInspectable var isLoading: Bool = false { didSet { applyStyle() } }
 	@IBInspectable var hasBorder: Bool = false { didSet { applyStyle() } }
-	// swiftlint:disable redundant_optional_initialization
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift:53]: You will have to re-add this to make the linter happy again :D

Comment (pascal-brause) [reply src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift:53]: Strange, a warning occurred that this does not trigger something. But now the linter wants it back....I rolled back my change.

**Pull request merged**